# HG changeset patch
# User Daniel J Sebald <daniel.sebald@ieee.org>
# Date 1373612717 18000
#      Fri Jul 12 02:05:17 2013 -0500
# Node ID ad92406ec915725095857f76a083765188c80b4b
# Parent  fb940d48c6f798751e3ef36e1e31e07dfbfb00cf
Add private files for image I/O to list of installed files

* scripts/image/module.mk
(image_PRIVATE_FCN_FILES): Add image/private/core_imfinfo.m, core_imread.m,
core_imwrite.m, imageIO.m, and imwrite_filename.m.

diff --git a/scripts/image/module.mk b/scripts/image/module.mk
--- a/scripts/image/module.mk
+++ b/scripts/image/module.mk
@@ -1,11 +1,16 @@
 FCN_FILE_DIRS += image
 
 image_PRIVATE_FCN_FILES = \
+  image/private/core_imfinfo.m \
+  image/private/core_imread.m \
+  image/private/core_imwrite.m \
+  image/private/imageIO.m \
+  image/private/imwrite_filename.m \
   image/private/ind2x.m
 
 image_FCN_FILES = \
   image/autumn.m \
   image/bone.m \
   image/brighten.m \
   image/cmpermute.m \
   image/cmunique.m \
